### PR TITLE
added check for data.state.lastchange

### PR DESCRIPTION
**Pull request by 0xphk**
module fails if lastchange is not existent in spaceapi .json data files and displays no status

### CODE REVIEW

File: py3status/modules/spaceapi.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def check(self, i3s_output_list, i3s_config):
             'cached_until': time() + self.cache_timeout
             }
 
-        try:
+        if True:
```
Comment (ultrabug) [inline py3status/modules/spaceapi.py]: I'm sorry mate but I don't understand this (useless?) "if" ?

**Comment by ultrabug** [comment]
Thanks @0xphk, I'm not sure I understand it all, I commented on the code.

Friendly ping to @gidsi.

**Comment by gidsi** [comment]
lastchange isn't required in the json file, thought it is:
http://spaceapi.net/documentation#documentation-ref-13-root-state-lastchange

there will be a few other bugs too (e.g. the short_text), will fix it. Don't understand the first "if" either.

**Comment by ultrabug** [comment]
Thanks @0xphk, it's been taken care of on PR #76 !

**Pull request closed without merging**